client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -130,15 +156,26 @@ float disp_z;
 	vec4 color;
 	// The alpha gives the ratio of sunlight in the incoming light.
 	float nightRatio = 1 - gl_Color.a;
-	color.rgb = gl_Color.rgb * (gl_Color.a * dayLight.rgb + 
+	color.rgb = gl_Color.rgb * (gl_Color.a * dayLight.rgb +
 		nightRatio * artificialLight.rgb) * 2;
 	color.a = 1;
-	
+
+#if defined(ENABLE_DIRECTIONAL_SHADING) && !LIGHT_EMISSIVE
+	// Lighting color
+	vec3 resultLightColor = ((lightColor.rgb * gl_Color.a) + nightRatio);
+
+	// ((resultLightColor * ((max(dot(normal, lightDirection), -0.2) + 0.2) / 1.2)* 0.6)) + 0.4;
+	resultLightColor = (resultLightColor * ((max(dot(alwaysNormal, lightDirection), -0.2) + 0.2) * 0.5)) + 0.4;
```

Comment:
Maybe this variable name is a little unclear? Should it refer to directional light somehow?